src/components/slider/slider.js

```diff
@@ -0,0 +1,210 @@
+import mixin from '../../globals/js/misc/mixin';
+import createComponent from '../../globals/js/mixins/create-component';
+import initComponentBySearch from '../../globals/js/mixins/init-component-by-search';
+import eventedState from '../../globals/js/mixins/evented-state';
+
+class Slider extends mixin(createComponent, initComponentBySearch, eventedState) {
+  /**
+   * Slider.
+   * @extends CreateComponent
+   * @extends InitComponentBySearch
+   * @param {HTMLElement} element The element working as an slider.
+   */
+  constructor(element, options) {
+    super(element, options);
+
+    this.sliderActive = false;
+    this.dragging = false;
+
+    this.track = this.element.querySelector(this.options.selectorTrack);
+    this.filledTrack = this.element.querySelector(this.options.selectorFilledTrack);
+    this.thumb = this.element.querySelector(this.options.selectorThumb);
+    this.input = this.element.querySelector(this.options.selectorInput);
+
+    if (this.element.dataset.sliderInputBox) {
+      this.boundInput = this.element.ownerDocument.querySelector(this.element.dataset.sliderInputBox);
+      this._updateInput();
+      this.boundInput.addEventListener('change', (evt) => { this._handleTextInput(evt); });
+    }
+
+    this._updatePosition();
+
+    this.thumb.addEventListener('mousedown', (evt) => { this._handleMouseDown(evt); });
+    this.element.ownerDocument.addEventListener('mouseup', (evt) => { this._handleMouseUp(evt); });
```

Comment:
This event listener should be released when slider instance is released. Same for the next line. See https://github.com/carbon-design-system/carbon-components/blob/51e0975/src/components/dropdown/dropdown.js#L29, etc for an example.